src/providers/BackgroundBlurProvider/docs/BackgroundBlurProvider.stories.mdx

```diff
@@ -35,7 +35,7 @@ before calling `createBackgroundBlurDevice`. Calling `createBackgroundBlurDevice
 `DefaultVideoTransformDevice` with `createBackgroundBlurDevice` in order to destroy the processors running previously. Once you call `DefaultVideoTransformDevice.stop`, you should discard the old `DefaultVideoTransformDevice`.
 For more information, refer to [Video Processing APIs](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/10_Video_Processor.md#stopping-videotransformdevice).
 
-One thing to note is that calling `meetingManager.selectVideoInputDevice()` with a `Device` type while the current selected video input device is a `VideoTransformDevice`
+One thing to note is that calling `meetingManager.startVideoInputDevice()` with a `Device` type while the current selected video input device is a `VideoTransformDevice`
```

Comment:
Now does the type needs to change from `Device`, may be `VideoInputDevice` now, just checking.

Reply:
No need to update for this sentence here. This one is talking about a specific case where the input device is of `Device` type.